### DISCUSSION TITLE
mmcls.utils 里面缺少 load_json_log组件

### DISCUSSION THREAD

**Discussion by PiPiZhuMaMa** [open-mmlab/mmclassification#1438] [Q&A]
安装环境是mmcv-2.0.0rc4 和mmclassification-1.x
模型训练可以正常进行
但是想可视化日志缺报错NameError: name 'load_json_log' is not defined
查看源码发现mmcls.utils里面已经没有load_json_log了
所有步骤均按照官方教程进行的https://mmclassification.readthedocs.io/zh_CN/dev-1.x/useful_tools/log_result_analysis.html
求解1.x版本应该如何进行日志的可视化？

**Comment by Ezra-Yu** [comment]
可视化日志 这个工具我们正在在1.x版本中支持，尽情期待
